pygmo/docstrings.cpp

```diff
@@ -1929,6 +1929,29 @@ See also the docs of the C++ class :cpp:class:`pagmo::dtlz`.
 )";
 }
 
+std::string wfg_docstring()
+{
+    return R"(__init__(prob_id = 1, dim_dvs = 5, dim_obj = 3, dim_k = 4)
+
+The WFG problem suite.
+
+Args:
+    prob_id (``int``): WFG problem id
```

Comment:
No reason to use the double quotes here, just write ``int``. We need to fix the other docstrings eventually.